### PR TITLE
Fix readme

### LINKED ISSUE #121: Think about minimal required knowledge to use art
Select list of tutorials that are useful and add them to readme.

### PR DESCRIPTION
**Pull request by kordc**
In this PR I'll try to close #121, and two bullets from #60.

### CODE REVIEW

File: art/experiment/experiment_state.py [hunk 1]
```diff
@@ -12,6 +11,7 @@ class ArtProjectState:
     step_states: Dict[str, Dict[str, Dict[str, str]]]
     status: str
     """
+    Singleton that knows the current experiment state + have information about the world around
```
Comment (trebacz626) [inline art/experiment/experiment_state.py]: It is not a singletone

**Comment by SebChw** [comment]
Just small issue that generally we would like people to install art with

`pip install art`

or something like this. This could be a new issue

**Comment by kordc** [comment]
> Just small issue that generally we would like people to install art with
> 
> `pip install art`
> 
> or something like this. This could be a new issue

Definitely agree, I was thinking about it. #132

File: art/experiment/experiment_state.py (hunk 1 above)
Comment (kordc) [reply art/experiment/experiment_state.py]: Ofc, it was just a leftover which I dummy copy passed into the doc string.

File: art/step/checks.py [hunk 2]
```diff
@@ -1,40 +1,104 @@
...
+        error (str): Error message if the check was not successful. Defaults to an empty string.
+    """
     is_positive: bool = field(default=True)
     error: str = field(default="")
 
 
 class Check(ABC):
-    name: str
-    description: str
-    required_files: List[str]
```
Comment (mmaecki) [inline art/step/checks.py:16]: Are the attributes removed on purpose? There is no __innit_ function.
They are described in the commend bellow them

Comment (kordc) [reply art/step/checks.py:16]: Good catch, probably a merge error. There's no init as it's an abstract class (ABC)

**Review by mmaecki — APPROVED** [review]

**Pull request merged**